firedrake/paraview_reordering.py

```diff
@@ -17,27 +17,30 @@
 and othe system information hence we fish out the exact name by trawling
 through the VTK library directory.
 """
-vtkSoLoc = importlib.util.find_spec("vtkmodules").submodule_search_locations[0]
-findStr = "vtkCommonDataModel"
-# Find the module name as this is system dependent in VTK9
-contents = os.listdir(vtkSoLoc)
-for item in contents:
-    if (findStr in item) and ("lib" not in item):
-        vtkSoName = "/" + item
-        break
-
-moduleName = "vtkCommonDataModel"
-loader = importlib.machinery.ExtensionFileLoader(moduleName,
-                                                 vtkSoLoc+vtkSoName)
-mod = loader.load_module(moduleName)
-
-vtkLagrangeTetra = mod.vtkLagrangeTetra
-vtkLagrangeHexahedron = mod.vtkLagrangeHexahedron
-vtkLagrangeTriangle = mod.vtkLagrangeTriangle
-vtkLagrangeQuadrilateral = mod.vtkLagrangeQuadrilateral
-vtkLagrangeWedge = mod.vtkLagrangeWedge
-
-paraviewUsesVTK8 = True
+try:
```

Comment:
Installing VTK in Spack is a pain. I can't get it to work without installing LLVM which takes a VERY long time. Maybe we could have our own minimal package? Either that or make it a Spack `variant`.

Obviously what I've done here is nonsense.

Reply:
report a bug to spack, this seems like a problem at their end.